src/ecs/commands/runCommandInContainer.ts

```diff
@@ -127,16 +48,34 @@ export async function runCommandInContainer(
             throw Error('SSM Plugin not installed and cannot auto install')
         }
 
-        const execCommand = await node.ecs.executeCommand(node.clusterArn, node.containerName, task, command)
+        const execCommand = await node.ecs.executeCommand(
+            node.clusterArn,
+            node.containerName,
+            response.task,
+            response.command
+        )
         const args = [JSON.stringify(execCommand.session), node.ecs.regionCode, 'StartSession']
+        showOutputMessage(
+            `${moment().format(LOCALIZED_DATE_FORMAT)}  Container: "${node.containerName}" Task ID: "${
+                response.task
+            }"  Running command: "${response.command}"`,
+            outputChannel
+        )
+
+        window.showInformationMessage(
+            localize(
+                'AWS.command.ecs.runCommandInContainer.runningCommandMessage',
+                'Running command. This may take several minutes.'
```

Comment:
Considerations:
* Add command name and container to message?
* "..." at the end instead of a period.
* Combined with feedback below, can we stream the output to the output channel and maybe provide a link here to open the output channel (as it is now, the output channel will open immediately, and switch back when the command completes...a button here would let the user swap away from the output channel and swap back)
  * If you do the above, consider outputting another notification on completion to signify the command is done with a link to the output channel.
* Consider converting to a progress (don't need to report specific progress increments, it's just nice to have the moving bar to show "progress" (also do indented bullet above if you swap to progress)

Reply:
>  Consider converting to a progress (don't need to report specific progress increments, it's just nice to have the moving bar to show "progress"

Yes. Separate PR though, please. This is in the backlog.